src/Compilers/Core/Portable/Operations/IFlowCaptureReferenceOperation.cs

```diff
@@ -9,8 +9,6 @@ namespace Microsoft.CodeAnalysis.FlowAnalysis
     /// </summary>
     public interface IFlowCaptureReferenceOperation : IOperation
     {
-        // PROTOTYPE(dataflow): Finalize the design how capturing/referencing intermediate results is represented.
```

Comment:
We decided to use `CaptureId`. @AlekseyTs let me know if this comment was capturing more work.